tests/wpt/metadata/animation-timing/same-dispatch-time.html.ini

```diff
@@ -0,0 +1,4 @@
+[same-dispatch-time.html]
+  type: testharness
+  [requestAnimationFrame will timestamp events in the same queue with the same time]
+    expected: PASS
```

Comment:
I think that `PASS` is the default expectation, and this file can be removed entirely.